packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.controller.js

```diff
@@ -94,6 +113,8 @@ export default /* @ngInject */
     }
   });
 
+  $scope.$on('ips.params.delete.ip', deleteServiceName);
```

Comment:
**nitpick:** This needs to be removed as well.

Reply:
Yep my bad I forgot it, it is done